include/mbedtls/error.h

```diff
@@ -127,6 +127,32 @@ extern "C" {
  */
 void mbedtls_strerror( int errnum, char *buffer, size_t buflen );
 
+/**
+ * \brief Translate high level part of a mbed TLS error code into a string
+ *        representation.
+ *
+ * This function returns a const pointer to an un-modifiable string. The caller
+ * must not try to modify the string and use it only for logging purposes.
```

Comment:
Why “use it only for logging purposes”?

Reply:
The intention here was to convey that these are un-modifiable strings and will mostly be used for logging purposes. Updated the doc to reflect the same.